setup.py

```diff
@@ -60,7 +61,7 @@
 REQUIREMENTS_DEPLOY = ["setuptools>=38.6.0", "twine>=1.11.0", "wheel>=0.31.0"]
 REQUIREMENTS_DEV = (
     [
-        "black==19.10b0",
+        "black",
```

Comment:
Do we not want to pin this at the current version? It seems to update quite erratically.

Reply:
I've only ever had one issue with black (the one that caused this pin). I'm tempted to leave it and see how it goes. If it breaks again, we can pin again.